src/guest_mem.S

```diff
@@ -60,3 +60,34 @@ _copy_from_guest:
 _ret_from_copy:
     mv    a0, t2
     ret
+
+// Fetch a guest instruction using HLVX. Only reads 2 or 4 byte instructions. Returns -1 on error.
```

Comment:
comment about the arguments expected to be passed for `_fetch_guest_instruction` would help.

Reply:
Done.